docs-devsite/ai.aioptions.md

```diff
@@ -22,14 +22,25 @@ export interface AIOptions
 
 |  Property | Type | Description |
 |  --- | --- | --- |
-|  [backend](./ai.aioptions.md#aioptionsbackend) | [Backend](./ai.backend.md#backend_class) | The backend configuration to use for the AI service instance. |
+|  [appCheck](./ai.aioptions.md#aioptionsappcheck) | [AppCheckOptions](./ai.appcheckoptions.md#appcheckoptions_interface) | Configures App Check usage for this AI service instance. |
+|  [backend](./ai.aioptions.md#aioptionsbackend) | [Backend](./ai.backend.md#backend_class) | The backend configuration to use for the AI service instance. Defaults to [GoogleAIBackend](./ai.googleaibackend.md#googleaibackend_class)<!-- -->. |
```

Comment:
I think that @dlarocque and I came up with a phrasing for how to refer to the API provider.  I can't seem to find it in the ref docs immediately, but maybe Daniel knows where it is?

Reply:
I took a stab at fitting that into this sentence, let me know if that works.